packages/cozy-harvest-lib/src/components/OAuthForm.jsx

```diff
@@ -72,30 +72,31 @@ export class OAuthForm extends PureComponent {
   }
 
   render() {
-    const { konnector, t, flowState } = this.props
-    const { initialValues, showOAuthWindow, needExtraParams, extraParams } =
-      this.state
+    const { konnector, t, flowState, reconnect, account } = this.props
+    const { showOAuthWindow, needExtraParams, extraParams } = this.state
     const isBusy =
       showOAuthWindow === true ||
       flowState.running ||
       (needExtraParams && !extraParams)
 
-    return initialValues ? null : (
+    return (
       <>
         <Button
           className="u-mt-1"
           busy={isBusy}
           disabled={isBusy}
           extension="full"
-          label={t('oauth.connect.label')}
+          label={t(`oauth.${reconnect ? 'reconnect' : 'connect'}.label`)}
```

Comment:
doesn't pass the grep test. http://jamie-wong.com/2013/07/12/grep-test/

This type of code will result in unused, unmaintained and other locales.